src/cincinnati/mod.rs

```diff
@@ -130,6 +132,59 @@ impl Cincinnati {
     }
 }
 
+/// Write MOTD indicating a dead-end release, with the passed `reason`.
+/// If `reason` is `None`, the MOTD written is left as an empty file.
+fn refresh_deadend_motd(reason: Option<String>) {
+    // Avoid showing partially-written messages using tempfile and
+    // persist (rename).
+    let f = tempfile::Builder::new()
+        .prefix("deadend")
```

Comment:
Nit: just for safety, I'd make this a dotfile.

Reply:
Done. Also added `.` on either side to make the tmpfile name more readable (`.deadend.abcdef.motd.partial` instead of `.deadendabcdefmotd.partial`) (not that it matters too much).